build_to_jdi-light.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -v
+
+# VARIABLES
+# Feel free to edit if your directory names or branches differ
+JDI_LIGHT_DIR="jdi-light"
+TEST_SITE_DIR=$(pwd)
+JDI_LIGHT_BRANCH="gh-pages"
+SERVER_PORT=8001
+
+echo "
+# This bash script displays the angular-site in browser for visual verification
```

Comment:
show by --help key, not always

Reply:
Done. Help is now displayed only if script is executed with -h or --help parameter